### PR TITLE
Add AppConfigDump API

### PR DESCRIPTION
**Pull request by atsui86**
@if6was9

### CODE REVIEW

File: src/main/java/io/macgyver/agent/sender/http/HttpAgentSender.java [hunk 1]
```diff
@@ -32,11 +32,13 @@
 	public static final String DEFAULT_CHECK_IN_PATH = "/api/cmdb/checkIn";
 	public static final String DEFAULT_APP_EVENT_PATH = "/api/cmdb/app-event";
 	public static final String DEFAULT_THREAD_DUMP_PATH = "/api/monitor/thread-dump";
+	public static final String DEFAULT_APP_CONFIG_DUMP_PATH = "/api/monitor/app-config-dump";
```
Comment (yaozhiqun) [inline src/main/java/io/macgyver/agent/sender/http/HttpAgentSender.java]: Can we simply use "/api/monitor/app-config"?

**Comment by yaozhiqun** [comment]
Looks good to me except a minor concern about simply using app config without dump suffix.

**Comment by if6was9** [comment]
I'll merge and publish this weekend. 

I want to just roll this code base into our internal bootstrap.  It is not useful as an open-source gizmo at this point.  

It did help force some structure at one point, but that point has passed for us.

**Comment by atsui86** [comment]
I'll merge this and roll it into bootstrap.  No need to publish.

**Pull request merged**